src/components/icons/index.ts

```diff
@@ -99,13 +99,20 @@ export { default as SliderOffIcon } from './slider-off'
 export { default as SliderOnIcon } from './slider-on'
 export { default as TaskStepDoneIcon } from './task-step-done'
 export { default as TickIcon } from './tick'
+export { default as ToolbarBlockquoteIcon } from './toolbar-blockquote'
 export { default as ToolbarBoldIcon } from './toolbar-bold'
+export { default as ToolbarBoxedTextIcon } from './toolbar-boxed-text'
 export { default as ToolbarCitationIcon } from './toolbar-citation'
 export { default as ToolbarCodeIcon } from './toolbar-code'
 export { default as ToolbarEquationIcon } from './toolbar-equation'
 export { default as ToolbarFigureIcon } from './toolbar-figure'
+export { default as ToolbarImageIcon } from './toolbar-image'
 export { default as ToolbarItalicIcon } from './toolbar-italic'
+export { default as ToolbarLinkIcon } from './toolbar-link'
+export { default as ToolbarMediaIcon } from './toolbar-media'
```

Comment:
There is already icon [OutlineEmbedIcon](https://github.com/Atypon-OpenSource/manuscripts-style-guide/blob/master/src/components/icons/outline-embed.tsx#L20) we use it for Draggable Tree, I think both should have the same standard icons